[DOCS] Remove all mentions of profilers in public docs (#8825)
great-expectations/great_expectations: docs/docusaurus/docs/terms/validator.md

When connecting to Data, it is often useful to verify that you have configured your <TechnicalTag relative="../" tag="datasource" text="Data Source" /> correctly.  To verify a new Data Source, you can load data from it into a Validator using a Batch Request.  Fore examples of this workflow, see [Connect to source data](../guides/connecting_to_your_data/connect_to_data_lp.md).

> When creating Expectations for an Expectation Suite, most workflows will have you use a Validator.  You can see this in our guide on how to create and edit Expectations with a DataAssistant.

Reviewer: ```suggestion
When creating Expectations for an Expectation Suite, you'll use a Validator for most workflows. For more information, see [Data Assistants](../guides/expectations/data_assistants_lp.md).
```
NOTE: Confirm link accuracy and functionality.
